grpc-gcp/src/test/java/com/google/grpc/gcp/SpannerIntegrationTest.java

```diff
@@ -238,7 +346,7 @@ public static void clearSessions() throws Exception {
   @Test
   public void testCreateAndGetSessionBlocking() throws Exception {
     SpannerBlockingStub stub = getSpannerBlockingStub();
-    CreateSessionRequest req = CreateSessionRequest.newBuilder().setDatabase(DATABASE).build();
+    CreateSessionRequest req = CreateSessionRequest.newBuilder().setDatabase(DATABASE_PATH).build();
```

Comment:
this line and the line above (SpannerBlockingStub stub = getSpannerBlockingStub())
could be included in setUp() method that is called before each test method.

Reply:
Actually some of the tests are not using blocking stub, they may use future stub, or async stub. So I think it would be better for each test method to initialize its own spanner stub.. WDYT?

Reply:
sgtm